package.json

```diff
@@ -20,6 +20,7 @@
     "build": "rollup -c",
     "build-min": "rollup -c --environment MINIFY:true",
     "prepublishOnly": "run-s build build-min",
+    "prepare": "run-s build build-min",
```

Comment:
Hi @danielsippel! Sorry for the delayed review. What's the purpose of adding the `prepare` script here?

Reply:
Hey @stepankuzmin !
When others npm install the mapbox-gl-draw dependency from e.g. my fork on github, the prepare script is run automatically by npm.. So the JS files are provided in /node_modules/dist/ automatically. Without the prepare script, the dist directory is empty after install. So forks can't be used without further manual build step.

Reply:
So you are currently using a fork with GitHub URL in your dependencies? We can probably safely drop this in the main repo, then.

Reply:
Right and yes.

Reply:
I see, thanks! Could you please drop it in the PR?